tests/integration/oauth_adapter_test.py

```diff
@@ -49,17 +49,21 @@ def authorize_with_providers(cls, oauth_adapters):
         redirect_uri = "http://local.broadinstitute.org/#fence-callback"
         state = "abc123"
         authz_responses = {}
+        # Whether this test is being run manually or for by automated integration.
+        # The integration run uses a fake provider. The manual run allows testing against real providers with real
+        # authentication. There's not a good way to run integration tests against real providers, so we have this
+        # flag to manually change the tests.
+        manual_run = False
         for provider, oauth_adapter in oauth_adapters.iteritems():
             authz_url = oauth_adapter.build_authz_url(scopes,
                                                       redirect_uri,
                                                       state,
-                                                      extra_authz_url_params={"idp": "google"})
+                                                      extra_authz_url_params={"idp": "google"} if manual_run else {"foo": "bar"})
```

Comment:
I think we can just leave this as is without the `if` statement all the time

Reply:
done